src/main/java/org/pac4j/dropwizard/Pac4jBundle.java

```diff
@@ -18,33 +26,51 @@
  * Dropwizard application's Jersey filter chain.
  */
 public abstract class Pac4jBundle<T extends Configuration>
-        implements ConfiguredBundle<T>, Pac4jConfiguration<T> {
+        implements ConfiguredBundle<T> {
     @Override
     public final void initialize(Bootstrap<?> bootstrap) {
-        // nothing to do
+        bootstrap.getObjectMapper().addMixIn(Client.class,
+                Pac4JMixins.ClientMixin.class);
+        bootstrap.getObjectMapper().addMixIn(BaseClient.class,
+                Pac4JMixins.BaseClientMixin.class);
+        bootstrap.getObjectMapper().addMixIn(Authenticator.class,
+                Pac4JMixins.AuthenticatorMixin.class);
+        bootstrap.getObjectMapper().addMixIn(PasswordEncoder.class,
+                Pac4JMixins.PasswordEncoderMixin.class);
+        bootstrap.getObjectMapper().addMixIn(CredentialsExtractor.class,
+                Pac4JMixins.CredentialsExtractorMixin.class);
+        bootstrap.getObjectMapper().addMixIn(ProfileCreator.class,
+                Pac4JMixins.ProfileCreatorMixin.class);
+        bootstrap.getObjectMapper().addMixIn(AuthorizationGenerator.class,
+                Pac4JMixins.AuthorizationGeneratorMixin.class);
+        bootstrap.getObjectMapper().addMixIn(Authorizer.class,
+                Pac4JMixins.AuthorizerMixin.class);
+        bootstrap.getObjectMapper().addMixIn(Matcher.class,
+                Pac4JMixins.MatcherMixin.class);
```

Comment:
So only these types will be available to create objects. In that case, the `RedirectActionBuilder` is missing as well as all types necessary for LDAP configuration (example: https://github.com/pac4j/pac4j/blob/master/pac4j-ldap/src/test/java/org/pac4j/ldap/test/tools/AuthenticatorGenerator.java#L19), MongoDB, SQL, but maybe it's too ambitious for a first version.